src/Microsoft.DotNet.Web.ProjectTemplates/content/EmptyWeb-FSharp/.template.config/vs-2017.3.host.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "name": {
+    "text": "Empty",
+    "package": "{0CD94836-1526-4E85-87D3-FB5274C5AFC9}",
+    "id": "1011"
+  },
```

Comment:
@phenning Where can I check that these properties (especially package and id) and "legacyTemplateIdentity" are correct for the three F# templates?

Reply:
`0CD94836-1526-4E85-87D3-FB5274C5AFC9` is the Id for the Razor template wizard Package.  It is defined [here](http://vstfdevdiv:8080/DevDiv2/DevDiv/_versionControl?path=%24%2FDevDiv%2FOffcycle%2FWPT%2FWebToolsExtensions%2FMain%2Fsrc%2FTemplateEngine%2FMicrosoft.VisualStudio.TemplateEngine.UI%2FDev15%2FMicrosoft.VisualStudio.TemplateEngine.pkgdef&version=T&_a=contents&line=1&lineStyle=plain&lineEnd=2&lineStartColumn=1&lineEndColumn=1)

Reply:
See my comment above in regards to the IDs. Technically the package that the package guid is referring to contains the base functionality for invoking all the template engine based templates, not just Razor.